examples/cim/dqn/config.yml

```diff
@@ -2,8 +2,15 @@ env:
   scenario: "cim"
   topology: "toy.4p_ssdd_l0.0"
   durations: 1120
-general:
+main_loop:
```

Comment:
learner_policy

Reply:
we need to refactor this part:
learner_policy
- main_loop
-- max_episode
-- early_stopping
- agent
-- rl_algorithms
--- exploration
-- learning_model
--- feature_engineer
--- learning_parameter

Reply:
need an offline discussion